--- origsrc/Nimesweeper-1.0/mine.h	2003-08-11 13:12:12.000000000 -0500
+++ src/Nimesweeper-1.0/mine.h	2008-09-03 17:22:29.446979200 -0500
@@ -27,7 +27,7 @@
 #define VERSION    ("1.0")
 #define DATE       ("17th November 2002")
 #define EMAIL      ("d4n13l@lycos.com")
-#define HIGHSCORES ("/usr/share/nimesweeper/HighScores.dat")
+#define HIGHSCORES ("/var/games/nimesweeper.scores")
 /* HIGHSCORES is a binary file, 40 items of HighScores,
    a top ten for each difficulty level */
 /* For HighScores structure */
